base/src/main/java/org/aya/distill/ConcreteDistiller.java

```diff
@@ -50,7 +50,7 @@ public ConcreteDistiller(@NotNull DistillerOptions options) {
   }
 
   @Override public Doc visitLam(Expr.@NotNull LamExpr expr, Outer outer) {
-    if (!options.showImplicitPats() && !expr.param().explicit()) {
+    if (!(boolean) options.map.get(DistillerOptions.Key.ShowImplicitPats) && !expr.param().explicit()) {
```

Comment:
is this cast necessary? (similar cases below and in `CoreDistiller`)

Reply:
??? wait what